io/src/csv/read_spacepoints.cpp

```diff
@@ -5,10 +5,12 @@
  * Mozilla Public License Version 2.0
  */
 
-// Local include(s).
-#include "read_spacepoints.hpp"
+// Project include(s).
+#include "traccc/definitions/primitives.hpp"
```

Comment:
It would be better to put this after the "local includes". Generally I try to go "from most local" to "most general" with the include order. (One can argue that this way more include mistakes can be found during the build.)

Reply:
The header sequence is rearranged